kroxylicious-test-tools/src/test/java/io/kroxylicious/proxy/config/ConfigurationTest.java

```diff
@@ -286,12 +289,192 @@ public static Stream<Arguments> fluentApiConfigYamlFidelity() {
 
     @ParameterizedTest(name = "{0}")
     @MethodSource
-    public void fluentApiConfigYamlFidelity(String name, Configuration config, String expected) throws Exception {
+    void fluentApiConfigYamlFidelity(String name, Configuration config, String expected) throws Exception {
```

Comment:
Noted. Thank you. I was under the misapprehension that public was required, so I'm made the same mistake in a few place across the test suite.  I'll raise a separate PR to fix the others.

Reply:
I installed the sonarlint plug-in. I've still got mixed feelings, I think it's overly pedantic and I'm not convinced there is a lot of value in changes such as this...

But going forward it makes sense as a default